src/Custom.Build.props

```
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <MinVerMinimumMajorMinor>5.4</MinVerMinimumMajorMinor>
+    <MinVerMinimumMajorMinor>5.6</MinVerMinimumMajorMinor>
```

@DavidBoike @bording Shouldn't master be bumped to 5.7?

```suggestion
    <MinVerMinimumMajorMinor>5.7</MinVerMinimumMajorMinor>
```